packages/at_secondary_server/lib/src/verb/handler/abstract_verb_handler.dart

```diff
@@ -161,6 +162,14 @@ abstract class AbstractVerbHandler implements VerbHandler {
     }
 
     final enrollNamespaces = enrollDataStoreValue.namespaces;
+    // set passed namespace. If passed namespace is null, get namespace from atKey
+    final keyNamespace =
+        namespace ?? (atKey != null ? AtKey.fromString(atKey).namespace : null);
+    if (keyNamespace == null && atKey == null) {
+      logger.shout('Both key and namespace are null');
+      return false;
+    }
```

Comment:
Should we do a check here : if both the key and namespace have been provided, and the key has a namespace, then throw an exception if the provided namespace and the key's namespace are different?

Reply:
done the changes

Reply:
added a new test class in tests to verify this..since we don't pass both atkey and namespace param from any of the verb handlers impl.